app/routes/profile.js

```diff
@@ -5,7 +5,7 @@ export default class ProfileRoute extends AuthenticatedRoute {
   @service router;
 
   canAccess() {
-    return this.abilities.can('show individual users');
+    return this.abilities.can('show own users');
```

Comment:
changing this is not strictly necessary, but it makes sense semantically to change it here too